modules/spotxBidAdapter.js

```diff
@@ -85,7 +85,7 @@ function Spotx() {
 
       bid.cpm = KVP_Object.spotx_bid;
       bid.vastUrl = url;
-      bid.ad = url;
+      bid.adId = KVP_Object.spotx_ad_key;
```

Comment:
This will break some internals in prebid.js unfortunately. Can you set the key using this method: 
https://github.com/prebid/Prebid.js/blob/master/modules/xhbBidAdapter.js#L23
Basically set another property on bid and retrieve it using that function. 

The only downside of overriding this key is you can't reference the bid later using this key but I assume that's ok.